sherpa/csrc/online_stream.cc

```diff
@@ -26,6 +26,7 @@
 #include "kaldifeat/csrc/feature-fbank.h"
 #include "kaldifeat/csrc/online-feature.h"
 #include "sherpa/csrc/hypothesis.h"
+#include "sherpa/csrc/endpoint.h"
```

Comment:
Please sort the incudes alphabetically.

If you are using vim, I suggest
https://github.com/google/vim-codefmt

It will format your code automatically on save if you have installed `clang-format`.

```bash
pip install clang-format==9.0.0
```